neo/public/javascripts/map/controllers.js

```diff
@@ -490,6 +490,30 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
       if ($scope.map)
         $scope.legend.addTo($scope.map);
 
+      //add the count of all the tweets control
+      var totalTweets = $('.totalTweets');
+
+      if (totalTweets){
+        console.log("drawMap: totalTweets: remove");
+        totalTweets.remove();
+      }
+
+      $scope.totalTweets = L.control({
+        position: 'bottomleft'
+      })
+
+      $scope.totalTweets.onAdd = function(map) {
+        var div = L.DomUtil.create('div', 'number legend');
+        div.style.margin = '0 0 30% 0';
+        div.innerHTML +=
+            '<h2>' + Asterix.totalCount + '</h2>' +
```

Comment:
The short answer is Yes. Right now since the middleware only sends hard-coded number, so if I change the number sent by the middleware, the front-end will display the new number.